### PR TITLE
Bump to Jekyll 1.0

### PR DESCRIPTION
**Pull request by skottler**

### CODE REVIEW

**Comment by domcleal** [comment]
Can you update the Rakefile too please so it runs the new command?

I think the config file will need updating too, as running `jekyll serve` says this:

```
   Deprecation: Auto-regeneration can no longer be set from your configuration file(s). Use the --watch/-w command-line option instead.
```

**Comment by skottler** [comment]
All set @domcleal.

**Comment by domcleal** [comment]
@skottler missing the Rakefile change?

**Comment by skottler** [comment]
@domcleal nope, the change doesn't need to be in the Rakefile. It's the removed `auto: true` in `_config.yml`.

**Comment by domcleal** [comment]
@skottler doesn't it need to change from --server etc?

File: Rakefile [hunk 1]
```diff
@@ -7,12 +7,12 @@ end
...
 task :build do
-  sh "jekyll"
+  sh "jekyll build"
 end
 
-desc 'Start server with --auto'
+desc 'Start server'
 task :server => :build do
-  sh "jekyll --server --auto"
+  sh "jekyll server"
```
Comment (domcleal) [inline Rakefile]: This needs -w or --watch to replace --auto.

File: Rakefile [hunk 2]
```diff
@@ -7,12 +7,12 @@ end
 
 desc 'Build site with Jekyll'
 task :build do
-  sh "jekyll"
+  sh "jekyll build"
 end
 
-desc 'Start server with --auto'
+desc 'Start server'
 task :server => :build do
```
Comment (domcleal) [inline Rakefile]: `jekyll server` appears to regenerate now on the first run, so this :build dependency should be removed I think, to save it running twice.

**Comment by domcleal** [comment]
Thanks @skottler, merged!

**Pull request merged**